packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -257,6 +259,46 @@ describe('Checkout', () => {
                 expect(spinner.exists()).toBe(true);
             });
         });
+
+        describe('nonFulfillableError', () => {
+            describe('when `nonFulfillableError.openInDialog` is `true`', () => {
+                it('should show a mega modal displaying the error title and description', () => {
+                    // Arrange
+                    const fulfilmentTimeIssue = CheckoutIssues[ERROR_CODE_FULFILMENT_TIME_UNAVAILABLE];
+                    const expectedErrorTitle = 'We do takeaways, not time machines';
+                    const expectedErrorMessage = 'The time you picked for your order isn\'t possible, please choose another.';
+
+                    // Act
+                    const wrapper = mount(VueCheckout, {
+                        i18n,
+                        store: createStore(),
+                        localVue,
+                        propsData,
+                        data () {
+                            return {
+                                nonFulfillableError: {
+                                    ...fulfilmentTimeIssue,
+                                    code: ERROR_CODE_FULFILMENT_TIME_UNAVAILABLE
+                                }
+                            };
+                        }
+                    });
+
+                    const errorModal = wrapper.find('[data-test-id="checkout-issue-modal"]');
+                    const errorTitle = wrapper.find('[data-test-id="checkout-issue-modal-title"]');
+                    const errorMessage = wrapper.find('[data-test-id="checkout-issue-modal-message"]');
+
+                    // Assert
+                    expect(errorModal.exists()).toBe(true);
+
+                    expect(errorTitle.exists()).toBe(true);
```

Comment:
Might not need the `exists` checks for some of these as your checking the `text()` which means the thing already exists.